### ISSUE TITLE
Update ESLint config dependencies

### ISSUE THREAD

**Issue by akxcv** [umbrellio/code-style#18]
`eslint-plugin-jsx-a11y` and `eslint-plugin-react` are out of date and should be updated.

**Comment by akxcv** [comment]
Closed via #21.
